client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+export const cardType = PropTypes.shape( {
+	card: PropTypes.string,
+	card_type: PropTypes.string,
+	name: PropTypes.string,
+	expiry: PropTypes.string,
+} );
+
+class StoredCard extends React.Component {
+	static propTypes = {
+		card: cardType,
+	};
+
+	render() {
+		const card = this.props.card;
+		const expirationDate = this.props.moment( card.expiry ).format( 'MM/YY' );
+		const cardClasses = classNames(
+			'credit-card__stored-card',
+			card.card_type && card.card_type.toLowerCase()
```

Comment:
card isn't required, so maybe
```jsx
get( card, [ 'card_type' ], '' ).toLowerCase()
```

Reply:
oh hmm on second reading see if the value matches an enum value and add a modifier class? for example `is-amex` or `is-visa`

Reply:
to further extend what @gwwar wrote - how about being super explicit so that someone coming around to this later knows what to reasonably expect?

```js
const cardType = card && card.card_type.toLocaleLowerCase();
classNames(
  'credit-card__stored-card',
  {
    'is-amex': cardType === 'amex',
    'is-mastercard': cardType === 'mastercard',
    'is-visa': cardType === 'visa'
  }
);
``